### PR TITLE
add privacy protection docs

### PR DESCRIPTION
**Pull request by zhouyun-zoe**

### CODE REVIEW

**Comment by kaikai1024** [comment]
忽略无关的文档

File: website/sidebars.json [hunk 1]
```diff
@@ -70,7 +79,7 @@
         "label": "Hacking",
         "type": "subcategory",
         "ids": [
-          "cita/hacking/compile-cita"
+          "cita/hacking/compile-cita",
```
Comment (kaikai1024) [inline website/sidebars.json]: Already fix it

**Comment by zhouyun-zoe** [comment]
> 忽略无关的文档

fixed

File: website/sidebars.json (hunk 1 above)
Comment (zhouyun-zoe) [reply website/sidebars.json]: > Already fix it

fixed

File: website/sidebars.json [hunk 2]
```diff
@@ -58,6 +58,15 @@
           "cita/account-permission/permission-example"
         ]
       },
+      {
+        "label": "Privacy Protection",
+        "type": "subcategory",
+        "ids": [
+          "cita/privacy/privacy-intro",
+          "cita/privacy/zktx",
```
Comment (kaikai1024) [inline website/sidebars.json:76]: delete the old zktx.

Comment (zhouyun-zoe) [reply website/sidebars.json:76]: > delete the old zktx.

fixed

**Review by kaikai1024 — DISMISSED** [review]

File: website/sidebars.json [hunk 3]
```diff
@@ -39,7 +39,7 @@
           "cita/system/snapshot",
           "cita/system/version",
           "cita/system/auto-exec",
-          "cita/system/zktx"
+          "cita/system/store"
```
Comment (kaikai1024) [inline website/sidebars.json]: delete it.

Comment (leeyr338) [inline website/sidebars.json]: You need to rebase this?

File: website/sidebars.json [hunk 4]
```diff
@@ -63,6 +63,15 @@
           "cita/account-permission/permission-example"
         ]
       },
+      {
+        "label": "Privacy Protection",
+        "type": "subcategory",
+        "ids": [
+          "cita/privacy/privacy-intro",
+          "cita/privacy/zktx",
+          "cita/privacy/crosschain-contract-example"
```
Comment (leeyr338) [inline website/sidebars.json:77]: You need to delete line 139?

Comment (zhouyun-zoe) [reply website/sidebars.json:77]: > You need to delete line 139?

fixed

File: website/sidebars.json (hunk 3 above)
Comment (zhouyun-zoe) [reply website/sidebars.json]: > You need to rebase this?

rebased

Comment (kaikai1024) [reply website/sidebars.json]: still

Comment (leeyr338) [reply website/sidebars.json]: Delete this line:
```
 "cita/system/store"
```

**Comment by kaikai1024** [comment]
I can help you to fix it....

**Review by kaikai1024 — APPROVED** [review]

**Pull request merged**